app/components/WoW/DeltaFilterControls.tsx

```
@@ -0,0 +1,161 @@
+                        break
+                      }
+                    }
+                  }
+
+                  newVisibleItems[itemName] = shouldBeVisible
+                }
+              )
+
+              onApplyFilter(newVisibleItems)
+            }}
+            className="mt-2 w-full bg-blue-500 hover:bg-blue-600 text-white text-xs font-medium py-1 px-2 rounded transition-colors">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)